#!/usr/bin/env python
PACKAGE = "first_project"

from dynamic_reconfigure.parameter_generator_catkin import * 

gen = ParameterGenerator()

frame_enum = gen.enum([gen.const("wheel_odom", int_t, 0, "wheel_odom frame"), 	
	gen.const("gps_odom", int_t, 1, "gps_odom frame")],
	"an enum to to set frame id for lidar data")

gen.add("target_frame_id", int_t, 0, "an enum to to set frame id for lidar data", 0, 0, 1, edit_method = frame_enum)

exit(gen.generate(PACKAGE, "lidar_remap", "parameters"))

